### PR TITLE
Add routes for static resources.

### PR DESCRIPTION
**Pull request by wwajerowicz**
Part of https://github.com/BlueBrain/nexus-kg/issues/304

### CODE REVIEW

File: modules/http/src/main/scala/ch/epfl/bluebrain/nexus/service/http/routes/StaticResourceRoutes.scala [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  private def folderContents(folder: File): Map[String, Json] = {
+    folder
+      .listFiles()
+      .toList
+      .filter(f => f.isFile && f.getName.endsWith(".json"))
+      .flatMap { file =>
+        parse(contentOf(file, baseReplacement)).toOption.map { json =>
+          file.getName.stripSuffix(".json") -> json
+      }
+    } toMap
```
Comment (hygt) [inline modules/http/src/main/scala/ch/epfl/bluebrain/nexus/service/http/routes/StaticResourceRoutes.scala]: I think we shouldn't abuse infix notation, much less postfix one.
https://docs.scala-lang.org/style/method-invocation.html#postfix-notation

File: modules/http/src/main/scala/ch/epfl/bluebrain/nexus/service/http/routes/StaticResourceRoutes.scala [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  }
+
+  private lazy val resources: Map[String, Map[String, Json]] = {
+    baseUri.toString()
```
Comment (bogdanromanx) [inline modules/http/src/main/scala/ch/epfl/bluebrain/nexus/service/http/routes/StaticResourceRoutes.scala]: missed this one...

File: modules/http/src/main/scala/ch/epfl/bluebrain/nexus/service/http/routes/StaticResourceRoutes.scala [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+import akka.http.scaladsl.model.{StatusCodes, Uri}
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import ch.epfl.bluebrain.nexus.commons.http.JsonLdCirceSupport._
+import io.circe.Json
+import io.circe.parser.parse
+
+import scala.io.Source
+
+class StaticResourceRoutes(resourcePath: String, prefix: String, baseUri: Uri) {
```
Comment (bogdanromanx) [inline modules/http/src/main/scala/ch/epfl/bluebrain/nexus/service/http/routes/StaticResourceRoutes.scala]: missing docs for reusable class

**Review by bogdanromanx — CHANGES_REQUESTED** [review]

File: modules/http/src/main/scala/ch/epfl/bluebrain/nexus/service/http/routes/StaticResourceRoutes.scala (hunk 1 above)
Comment (bogdanromanx) [reply modules/http/src/main/scala/ch/epfl/bluebrain/nexus/service/http/routes/StaticResourceRoutes.scala]: +1 on this one

**Review by hygt — DISMISSED** [review]

**Review by bogdanromanx — APPROVED** [review]

**Pull request merged**